Assets/BossRoom/Scripts/Server/Game/State/ServerCharSelectState.cs

```diff
@@ -26,7 +26,13 @@ private void OnClientChangedSeat(ulong clientId, int newSeatIdx, bool lockedIn)
         {
             int idx = FindLobbyPlayerIdx(clientId);
             if (idx == -1)
-                throw new System.Exception($"OnClientChangedSeat: client ID {clientId} is not a lobby player and cannot change seats!");
+            {
+                //FIXME:MLAPI See note about MLAPI issue 745 in CoroSeatNewPlayer.
```

Comment:
Currently checking with Will to get the internal jira ticket for this, we should track this to make sure we fix this once it's fixed MLAPI side.

Reply:
Noted. Noel has told me the real fix for this isn't coming until 1.0, which I understand is probably still a ways out.